test/terraform/gke/main.tf

```diff
@@ -1,4 +1,4 @@
-provider "google" {
+provider "google-beta" {
   project = var.project
```

Comment:
Should we set the specific provider version?

Reply:
Hmm I don't know if we need to, so I'd lean towards keeping it as is until we see problems.

Reply:
If this is a beta though then it might often have breaking changes.

Reply:
Do we have resolution on this?

Reply:
Yep, added the version to be the latest.